src/inc/msquic.h

```diff
@@ -319,6 +320,7 @@ typedef struct QUIC_CREDENTIAL_CONFIG {
     void* Reserved; // Currently unused
     QUIC_CREDENTIAL_LOAD_COMPLETE_HANDLER AsyncHandler; // Optional
     QUIC_ALLOWED_CIPHER_SUITE_FLAGS AllowedCipherSuites;// Optional
+    const char* CaCertificateFile;
```

Comment:
Two things:

1. This should have a comment indicating it's optional.
2. You can actually leverage the `Reserved` space instead of appending.